.docker/Dockerfile.php

```diff
@@ -1,11 +1,14 @@
 # @see https://github.com/dpc-sdp/bay/blob/master/bay/images/Dockerfile.php
 ARG CLI_IMAGE
+ARG BAY_IMAGE_VERSION=latest
+
 FROM ${CLI_IMAGE:-cli} as cli
 
-FROM singledigital/bay-php:latest
+FROM singledigital/bay-php:${BAY_IMAGE_VERSION}
 
 # Antivirus update returns non-zero codes.
 # @see https://github.com/clamwin/clamav/blob/0.100.1/freshclam/freshclamcodes.h#L23
-RUN  freshclam --no-warnings || true
+RUN apk --update add clamav clamav-libunrar \
+&& freshclam --no-warnings || true
```

Comment:
We should look to create a ticket to centralise this service in Lagoon, we can have Drupal connect to ClamAV via TCP and then manage a single virus definition with a long running process.

This would have a few benefits:
- Reduce the size of the image
- Improve build time of the image (eg. freshclam can hang)